index.bs

```diff
@@ -349,6 +349,220 @@ One example of a feature that should be limited to secure contexts
 is geolocation, since the authentication and confidentiality provided by
 secure contexts reduce the risks to user privacy.
 
+<h2 id="css">Cascading Style Sheets (CSS)</h2>
+
+<h3 id="css-versus-markup">Presentation should be in CSS, content should be in HTML</h3>
+
+The separation between CSS and HTML was originally designed around the idea
+that the HTML is for the *content* and
+the CSS is for the *presentation* of that content.
+The CSS was intended to be optional,
+and the HTML intended so that it could be fully understood without the CSS.
+While the idea that the CSS can be dropped and the markup can still be usable
+may not make sense for *applications* built using web technologies,
+it still seems like a separation worth maintaining for *documents*,
+which remain a major use of web technology.
+
```

Comment:
Separation of content and presentation matters for applications also. It might be much harder to understand an app without any styling, because it requires a richer semantic vocabulary than base-level HTML does, but it does need to be understandable without CSS for non-visual UAs, and alternate styling should be possible on top of the same base markup. I don't think features designed for apps should be given a free pass on this point.

Reply:
As discussed in today's teleconference, I'm going to drop this section and merge the rest.  Landing a replacement for this text is covered in #169.